PowerAux/src/main.cpp

```diff
@@ -13,20 +13,18 @@
 
 BufferedSerial device(USBTX, USBRX);
 
-CAN bms_can(CAN_RX, CAN_TX);
 DigitalOut bms_can_stby(CAN_STBY);
-Thread bms_can_tx_thread;
 Thread bms_can_rx_thread;
+Thread bms_can_tx_thread;
 
-CAN vehicle_can(CAN2_RX, CAN2_TX);
 DigitalOut vehicle_can_stby(CAN2_STBY);
-Thread vehicle_can_tx_thread;
 Thread vehicle_can_rx_thread;
+Thread vehicle_can_tx_thread;
 
 PowerAuxCANParser bms_can_parser;
 PowerAuxCANParser vehicle_can_parser;
-CANInterface bms_can_interface(bms_can, bms_can_parser, bms_can_tx_thread, bms_can_rx_thread, &bms_can_stby, CAN_PERIOD);
-CANInterface vehicle_can_interface(vehicle_can, vehicle_can_parser, vehicle_can_tx_thread, vehicle_can_rx_thread, &vehicle_can_stby, CAN_PERIOD);
+CANInterface bms_can_interface(CAN_RX, CAN_TX, bms_can_parser, bms_can_rx_thread, bms_can_tx_thread, &bms_can_stby, CAN_PERIOD);
+CANInterface vehicle_can_interface(CAN2_RX, CAN2_TX, vehicle_can_parser, vehicle_can_rx_thread, vehicle_can_tx_thread, &vehicle_can_stby, CAN_PERIOD);
```

Comment:
can we continue with the BMS_CAN/VEHICLE_CAN scheme here as opposed to CAN_xx/CAN2_xx? I feel like it's more descriptive.